deep-security-agent/tasks/check-in-with-manager.yml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2018, Trend Micro
+#
+# License as per [repo](master/LICENSE)
+#
+# *********************************************************************
+# * Ask the Deep Security agent to check-in
+# *********************************************************************
+---
+# *********************************************************************
+# * Check if DSA installed properly
```

Comment:
The check if DSA installed properly seems to be reused a lot, maybe we can have a separated yml to reduce the duplicated and make these function focus on the command it self.

Reply:
Even better, if possible, we can simply compose the parameter to another yml, so all these actions will be a very simple yml itself.

Reply:
Refactor the rest of 6 tasks into 1. Modified related README.md